projects/ngx-prx-styleguide/src/lib/datepicker/datepicker.component.ts

```diff
@@ -72,7 +77,9 @@ export class DatepickerComponent implements AfterViewInit {
   picker: Pikaday;
 
   dateDefinedAndChanged(value: Date) {
-    return value && (!this._date || this._date.valueOf() !== value.valueOf());
+    return value &&
+      /* !Invalid Date */ !isNaN(value.valueOf()) &&
```

Comment:
JavaScript Invalid Date, ran into this with a `moment` initialized with null from the tz picker